content/ros/ROS Introduction/presentation/ROS_Introduction.tex

```diff
@@ -0,0 +1,902 @@
+\documentclass{beamer}
+
+
+\usepackage[utf8]{inputenc}
+\usepackage{amsmath}
+\usepackage{amsfonts}
+\usepackage{amssymb}
+\usepackage{graphicx}
+\usepackage{ragged2e}  % `\justifying` text
+\usepackage{booktabs}  % Tables
+\usepackage{tabularx}
+\usepackage{tikz}      % Diagrams
+\usetikzlibrary{calc, shapes, backgrounds}
+\usepackage{amsmath}
+\usepackage{amssymb}
+\usepackage{dsfont}
+\usepackage{url}       % `\url
+\usepackage{listings}  % Code listings
+\usepackage[T1]{fontenc}
+\usepackage[percent]{overpic}
+\usetikzlibrary{trees}
+\usepackage[absolute,overlay]{textpos}
+
+\usepackage{theme/beamerthemehbrs}
+
+\author[MAS]{Hassan Umari}
+\title{Introduction to ROS}
+\subtitle{Foundation Course}
+\institute[HBRS]{Hochschule Bonn-Rhein-Sieg}
+\date{\today}
+\subject{ROS workshop}
+
+% \thirdpartylogo{path/to/your/image}
+
+
+\begin{document}
+{
+\begin{frame}
+\titlepage
+\end{frame}
+}
+
+
+\section{What is ROS?}
+
+
+\subsection{What ROS is}
+\begin{frame}{What ROS is}
+\framesubtitle{Robot Operating System}
+    \begin{itemize}
+        \item Short for: Robot Operating System.
+        \item A collection of libraries and tools.
+        \item It helps software developers create robot
+        applications. 
+    \end{itemize}
+\end{frame}
+
+
+
+\begin{frame}[plain]{}
+    \centering
+    \includegraphics[width=.66\linewidth]{figures/stop_reinventing_theWheel.jpg}
+    
+\tiny{http://www.willowgarage.com/blog/2010/04/27/reinventing-wheel}
+\end{frame}
+
+
+\begin{frame}{What ROS is}
```

Comment:
Another very important item, and perhaps one of the main advantages of ROS, is that it makes it possible to write components in different programming languages and integrate them together fairly easily.

Reply:
I was going to mention this in ROS features section, in the language independent slide, so during the discussion not explicitly on the slide.